addons/web/static/src/views/form/form_controller.scss

```diff
@@ -981,20 +982,22 @@
     .o_form_view {
         min-height: auto;
 
-        --formView-sheetBg-padding-top: #{map-get($spacers, 2)};
-        --formView-sheetBg-padding-x: #{$modal-inner-padding};
+        .o_form_sheet_bg {
+            --formView-sheetBg-padding-top: 0;
```

Comment:
Looks like we regress a bit here, if you go to Time Off > Overview > New

We don't have a `padding-top` anymore inside the modal so the buttons stick to the border, could you have a look?

**saas-16.3 :**
![image](https://github.com/odoo/odoo/assets/128030743/86bbb4bd-8687-4e8e-b873-d17b1a50e79f)

**saas-16.3-modal-fixes-edi :** 
![image](https://github.com/odoo/odoo/assets/128030743/0cea56c4-6ece-4c5d-90d7-b9f4ab2b73e9)